### PR TITLE
Feat: Allow customisation of the hacky submit selector, this allows inline actions to work with CMSEditPage and an extension.

### PR DESCRIPTION
**Pull request by LiamKearn**
As the title says, its not pretty but it works :)

### CODE REVIEW

File: src/CmsInlineFormAction.php [hunk 1]
```diff
@@ -103,6 +117,9 @@ public function FieldHolder($properties = array())
         if ($this->readonly) {
             $attrs .= ' style="display:none"';
         }
+        if (\strlen($this->submitSelector)) {
```
Comment (LiamKearn) [inline src/CmsInlineFormAction.php:120]: Wrong project settings my bad!
```suggestion
        if (strlen($this->submitSelector)) {
```

File: src/CmsInlineFormAction.php [hunk 2]
```diff
@@ -162,4 +179,28 @@ public function setPost($post)
...
+     */
+    public function getSubmitSelector()
+    {
+        return $this->submitSelector;
+    }
+
+    /**
+     * Set the value of {@see self::$submitSelector}
+     * 
+     * Includes
```
Comment (LiamKearn) [inline src/CmsInlineFormAction.php:196]:
```suggestion
     * Includes the full selector, Anything that flys as a Jquery selector.
```

**Comment by LiamKearn** [comment]
Thanks for the module :)

**Comment by lekoala** [comment]
@LiamKearn thanks for the PR ;)

**Pull request merged**